.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*	@xataio/frontenders
```

Comment:
This is giving an error in the PR diff. I was thinking that maybe the team is secret but we have this in its configuration

<img width="524" alt="image" src="https://user-images.githubusercontent.com/50486/156534871-56cbc27b-65b2-48de-a871-e4b7ecef8a03.png">

🤔

Reply:
Does the team have write access?

Reply:
That was it. Thanks!